extensions/azurePublish/src/components/util.tsx

```diff
@@ -0,0 +1,100 @@
+import { getTenants, getARMTokenForTenant } from '@bfc/extension-client';
+import jwtDecode from 'jwt-decode';
+import { SharedColors } from '@uifabric/fluent-theme';
+import { AzureResourceTypes } from '../types';
+
+export function decodeToken(token: string) {

```

Comment:
nit: is there any particular reason that some of the functions in this file are arrow and some are using traditional function definition? (prefer arrow)